addons/mass_mailing/models/mailing_contact.py

```diff
@@ -117,3 +117,10 @@ def _message_get_default_recipients(self):
             'email_cc': False}
             for r in self
         }
+
+    @api.model
+    def fields_get(self, fields=None, attributes=None):
```

Comment:
could be cool to explain in a comment why you remove opt_out from the fields_get.